### PR TITLE
Issue01 - Item Card Component

### LINKED ISSUE #1: 01 - Item Card Component
As a user, I want to be able to see donated items with relevant information attached to them.

Create a frontend component of an Item card that can be listed on the main page of the app. You do not need to create the tags in the task (it will be a separate task).

- [ ] View the design on [Figma](https://www.figma.com/file/soYIk25VmoUkM3ha3WFEMV/Willshire-Community---H4I?node-id=0%3A1&t=UxasY5wHdq3w7Y7D-0)
- [ ] Create the component with generic data
- [ ] Have the component take in props for the item name, date, money, and categories (tags)

Screenshot
image

Notes:
- any styles you think should be global, put in app.css
- choose date format that is easiest to read. likely dd/mm/yyyy

### PR DESCRIPTION
**Pull request by andrewtchan**
### Pull Request Summary

Submitting pull request for Issue 01 including ItemCard.tsx and ItemCard.css. Closes #1 

### Pull Request Checklist

- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [x] The developer name is specified
- [x] The summary is completed
- [x] Ask for a review in communication channels

### Reviewer Checklist - PULL REQUEST REVIEWER ONLY

IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized pull request reviewers. If you are the individual contributor, do not fill out the rest of the fields or check the boxes.

## Reviewer: Full Name

NOTE: Pull requests should only be merged when all boxes are checked.

- [x] Assign yourself as reviewer if not assigned
- [x] The reviewer name is specified
- [x] The code is fully reviewed
- [x] Meaningful feedback is given
- [x] Let developer know a review has been made

### CODE REVIEW

**Comment by jlaksana** [comment]
Please undo all the modifications to the other files that you didn't create. This is probably left over stuff from when we did setup

**Comment by andrewtchan** [comment]
Reverted all changes to files besides ItemCard.tsx and ItemCard.css

**Review by jlaksana — CHANGES_REQUESTED** [review]

File: src/components/ItemCard.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.item-card {
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (andrewtchan) [reply src/components/ItemCard.css:2]: I think the absolutes for the nested divs within the item-card div are necessary for spacing within the ItemCard. I looked at the css documentation and I think just removing the positioning attributes from the top level class (like in my commit) does what you wanted, which is make the component track the top left. Let me know if that's still not correct.

File: src/components/ItemCard.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+export default function ItemCard({ itemName, date, money }: ItemCardProps) {
+  return (
+    <div className="item-card">
+      <div className="item-card-imgbox" />
+      <div className="item-card-info">
+        <div className="item-titlebox">
+          <div className="item-card-name">{itemName}</div>
+          <div className="item-card-date">{date}</div>
```
[the remark this answers is not in the document]
Comment (andrewtchan) [reply src/components/ItemCard.tsx]: I used mm/dd/yyyy formatting for this.

**Review by jlaksana — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**